### PR TITLE
Task 9/rx

### PR DESCRIPTION
**Pull request by captain2b**

### CODE REVIEW

File: src/app/shared/search/search.component.ts [hunk 1]
```diff
@@ -7,14 +9,27 @@ import { Component, EventEmitter, OnInit, Output } from '@angular/core';
 })
 export class SearchComponent implements OnInit {
   searchText: string;
+  searchSubject$ = new Subject<string>();
   @Output() someEvent: any = new EventEmitter<any>();
```
Comment (dKab) [inline src/app/shared/search/search.component.ts]: funny name

File: src/app/services/loader.service.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  constructor() {
+  }
+
+  show() {
+    this.loaderSubject.next(<any>{ show: true });
+  }
+
+  hide() {
+    setTimeout(() => {
```
Comment (dKab) [inline src/app/services/loader.service.ts:20]: why do you need Timeout here? Is it because requests are too fast and otherwise it blinks too quick?

**Review by dKab — COMMENTED** [review]
I don't see user info in the header. Is it me, or did you forget to implement it?

**Review by dKab — APPROVED** [review]

File: src/app/services/loader.service.ts (hunk 2 above)
Comment (captain2b) [reply src/app/services/loader.service.ts:20]: yes, its annoying.  Should i delete it?

File: src/app/shared/search/search.component.ts (hunk 1 above)
Comment (captain2b) [reply src/app/shared/search/search.component.ts]: name changed

File: src/app/services/loader.service.ts (hunk 2 above)
Comment (dKab) [reply src/app/services/loader.service.ts:20]: It's up to you 😃

**Pull request merged**